multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/jobs/CleanUpJob.java

```diff
@@ -31,8 +30,11 @@
     List<Cleaner> cleaners;
     private final SafeExecutor safeExecutor = new SafeExecutor(CleanUpJob::log);
 
-    @Override
-    public void execute(JobExecutionContext context) {
+    @Scheduled(cron = "#{@applicationConfiguration.getCronExpressionForOldData()}")
```

Comment:
applicationConfiguration, but the name of the field in the class is configuration. Does spring resolve this as well?

Reply:
It resolves the name in the same way it resolves injected objects. Actually it does not reference the field, but rather injects ApplicationConfiguration again, because the value in the annotation is static.